rt/rs/extensions/rx2/src/main/java/org/apache/cxf/jaxrs/rx2/client/FlowableRxInvokerImpl.java

```diff
@@ -147,34 +150,90 @@ public FlowableRxInvokerImpl(WebClient wc, ExecutorService ex) {
 
     @Override
     public <T> Flowable<T> method(String name, Entity<?> entity, Class<T> responseType) {
+        
+        Flowable<T> flowable = Flowable.create(new FlowableOnSubscribe<T>() {
+            @Override
+            public void subscribe(FlowableEmitter<T> emitter) throws Exception {
+                    try {
+                        T response = syncInvoker.method(name, entity, responseType);
+                        emitter.onNext(response);
```

Comment:
@jkoehler22  Do you think we should check if emitter is cancelled before calling oNext / etc?
```
if (!emitter.isCancelled()) {
       emitter.onNext(response);
}
```

Reply:
Agree.  Thanks for the comments.  Done.